build.gradle

```diff
@@ -22,6 +22,7 @@ if (!repositories) {
       }
     }
     repositories {
+      mavenCentral()
```

Comment:
Thanks for testing it @Scottmitch,
I've opened a separate PR for visibility #2215.